hazelcast/src/main/java/com/hazelcast/multimap/impl/operations/MergeBackupOperation.java

```diff
@@ -62,9 +62,12 @@ public void run() throws Exception {
             } else {
                 MultiMapValue containerValue = container.getOrCreateMultiMapValue(key);
                 Collection<MultiMapRecord> collection = containerValue.getCollection(false);
+                container.decrementSize(collection.size());
```

Comment:
We don't want everyone messing with the size of the container from the outside. This should be taken care of by the internals of the map container. Otherwise we end up with procedural spaghetti.